recipe/meta.yaml

```diff
@@ -30,27 +26,35 @@ requirements:
     - libgcrypt
     - libgpg-error
     - libiconv  # [not linux]
-    - libtool
-    - libxml2
-    - libxslt
+    - libtool    
+    - libxml2 2.10
+    - libxslt 1.1.37
     - openssl
```

Comment:
that is C/C++ shouldn't we have explicit pinning ?

Reply:
Added